providers/session_state.go

```diff
@@ -62,6 +62,19 @@ func (s *SessionState) EncodeSessionState(c *cookie.Cipher) (string, error) {
 	} else {
 		ss = *s
 		var err error
+		// Encrypt also Email and User when cipher is provided
```

Comment:
I'm not sure this comment is necessary, personally I would drop it but happy to leave if you feel strongly about it

Reply:
Yes, I guess from this point forward there is no different treatment in encrypting the email and user versus the other fields. Removing the comment.